.travis.yml

```diff
@@ -40,7 +42,9 @@ before_script:
   - composer update --prefer-source --no-interaction $COMPOSER_FLAGS
   - sudo pip install -r Resources/doc/requirements.txt
 
-script: make test
+script:
+ - make test
+ - if [ "$CS_FIXER" = "run" ]; then make cs_dry_run ; fi;
```

Comment:
At least, before `make test` but not on `before_script`.

IMO, code review it's a test finally.

Is Travis would be failed if an cs fixer return `1` on `before_script` section?